packages/framework/ai-collab/src/test/explicit-strategy/agentEditing.spec.ts

```diff
@@ -0,0 +1,271 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+
+// eslint-disable-next-line import/no-internal-modules
+import { createIdCompressor } from "@fluidframework/id-compressor/internal";
+// eslint-disable-next-line import/no-internal-modules
+import { MockFluidDataStoreRuntime } from "@fluidframework/test-runtime-utils/internal";
+import {
+	SchemaFactory,
+	getJsonSchema,
+	SharedTree,
+	TreeViewConfiguration,
+	// eslint-disable-next-line import/no-internal-modules
+} from "@fluidframework/tree/internal";
+// eslint-disable-next-line import/no-internal-modules
+import type { ResponseFormatJSONSchema } from "openai/resources/shared.mjs";
+
+// eslint-disable-next-line import/no-internal-modules
+import { objectIdKey } from "../../explicit-strategy/agentEditTypes.js";
+// eslint-disable-next-line import/no-internal-modules
+import { IdGenerator } from "../../explicit-strategy/idGenerator.js";
+// import { getResponse } from "../../explicit-strategy/llmClient.js";
```

Comment:
```suggestion
```

Reply:
Fixed in commit bac309434408ff1fd2adfbc0dcacf19c89f40ee8